sklearn/cross_decomposition/pls_.py

```diff
@@ -620,8 +660,33 @@ class PLSCanonical(_PLS):
 
     Notes
     -----
+    Matrices :
+        T: x_scores_
+        U: y_scores_
+        W: x_weights_
+        C: y_weights_
+        P: x_loadings_
+        Q: y_loadings__
+
+    Are computed such that:
+        X = T P.T + Err and Y = U Q.T + Err
+        T[:, k] = Xk W[:, k] for k in range(n_components)
+        U[:, k] = Yk C[:, k] for k in range(n_components)
+        x_rotations_ = W (P.T W)^(-1)
+        y_rotations_ = C (Q.T C)^(-1)
+    where Xk and Yk are residual matrices at iteration k.
+
+    Slides explaining PLS
+    :ref:http://www.eigenvector.com/Docs/Wise_pls_properties.pdf
+
+<<<<<<< HEAD
+    For each component k, find weights u, v that optimizes:
+    ``max corr(Xk W[:, k], Yk C[:, k]) * var(Xk W[:, k]) var(Yk C[:, k])``,
+     such that ``|C[:, k]| = 1``
+=======
     For each component k, find weights u, v that optimize::
-    max corr(Xk u, Yk v) * var(Xk u) var(Yk u), such that ``|u| = |v| = 1``
+    max corr(Xk u, Yk v) * std(Xk u) std(Yk u), such that ``|u| = |v| = 1``
+>>>>>>> Adressed comments
```

Comment:
@kastnerkyle this is what I meant wrt var/std.

But conflict needs to be resolved